### ISSUE TITLE
Need function `learning_rate_plot()` which can be used to tune model by user

### ISSUE THREAD

**Issue by millerh1** [dmontemayor/jdrfmetabo#23]
We need a function `learning_rate_plot()` which provides a user with a plot for diagnosing the learning rate. We also need directions in the vignette that shows the user how to use this to tune their model.

**Comment by millerh1** [comment]
Implemented with `autoPLIER.plot`
